package.json

```diff
@@ -2,18 +2,22 @@
   "name": "xterm-pty",
   "description": "A pty for xterm.js and Emscripten",
   "version": "0.10.2",
-  "main": "index.js",
+  "main": "out/index.js",
+  "module": "out/index.mjs",
+  "types": "out/index.d.ts",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/mame/xterm-pty.git"
   },
   "license": "MIT",
   "scripts": {
-    "build": "tsc && webpack"
+    "build": "tsup && webpack"
   },
   "files": [
+    "index.js",
     "index.js.map",
-    "emscripten-pty.js"
+    "emscripten-pty.js",
+    "out/**.*"
```

Comment:
I don't want to include everything in `out` directory.
Can you only specify `index.js`, `index.js.map`, and `index.d.ts` explicitly? If you need `index.mjs`, you can include that too.